README.md

```diff
@@ -5,3 +5,5 @@
 [![license](https://img.shields.io/github/license/pydantic/pydantic-settings.svg)](https://github.com/pydantic/pydantic-settings/blob/main/LICENSE)
 
 **Work in Progress**, see https://github.com/pydantic/pydantic/pull/4492
+
+{!docs/index.md!}
```

Comment:
I don't think we should include this, since it won't work in github.

Reply:
My plan was to make new docs available for the users because the old docs are outdated and we need to somehow share the new docs with the user.